.golangci.yml

```diff
@@ -0,0 +1,38 @@
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 20m
+
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
```

Comment:
Do we believe in all these options?

Reply:
Yeah these are the same options that we are using on all of our other repos